lib/delayed/performable_method.rb

```diff
@@ -22,18 +22,42 @@ def display_name
       end
     end
 
-    def perform
-      object.send(method_name, *args) if object
+    if RUBY_VERSION >= '3.0'
+      def perform
+        if args_is_a_hash?
+          object.send(method_name, **args.first)
+        else
+          object.send(method_name, *args)
+        end if object
```

Comment:
This fails if the only argument is intentionally a hash, for example sending partial request params to a job.